mbf_abstract_nav/src/move_base_action.cpp

```diff
@@ -220,8 +224,8 @@ void MoveBaseAction::actionExePathFeedback(
     {
       std::stringstream oscillation_msgs;
       oscillation_msgs << "Robot is oscillating for " << (ros::Time::now() - last_oscillation_reset_).toSec() << "s!";
-      ROS_WARN_STREAM_NAMED("exe_path", oscillation_msgs.str());
-      action_client_exe_path_.cancelGoal();
+      ROS_WARN_STREAM_NAMED("move_base", oscillation_msgs.str());
+      cancelActiveClients();
```

Comment:
There is no other client running at that time, or am I wrong?

Reply:
True; reverted